gradle/libs.versions.toml

```diff
@@ -1,5 +1,9 @@
 [versions]
 
+# Core Library Desugaring
+# https://github.com/google/desugar_jdk_libs/blob/master/CHANGELOG.md.
+desugaring = "1.1.5"
```

Comment:
There's actually a newer version of the desugaring library available as of the time of writing: [2.0.0](https://github.com/google/desugar_jdk_libs/blob/master/CHANGELOG.md)

Reply:
I have to update AGP first I believe, this will come along with it in a consequent PR.